install/generator/09-syndesis-public-oauth-proxy.yml.mustache

```diff
@@ -0,0 +1,177 @@
+      spec:
+        containers:
+        - name: syndesis-public-oauthproxy
+{{^WithDockerImages}}
+          image: ' '{{/WithDockerImages}}{{#WithDockerImages}}
+          image: ${SYNDESIS_REGISTRY}/{{Images.OAuthProxyImagePrefix}}/{{ Images.Support.OAuthProxy }}:{{ Tags.OAuthProxy }}{{/WithDockerImages}}
+          args:
+            - --provider=openshift
+{{^WithOAuthClient}}
+            - --client-id=system:serviceaccount:${OPENSHIFT_PROJECT}:syndesis-oauth-client{{/WithOAuthClient}}{{#WithOAuthClient}}
+            - --client-id=syndesis-${OPENSHIFT_PROJECT}{{/WithOAuthClient}}
+            - --client-secret=${OPENSHIFT_OAUTH_CLIENT_SECRET}
+            - --upstream=http://syndesis-server/api/v1/public/
+            - --tls-cert=/etc/tls/private/tls.crt
+            - --tls-key=/etc/tls/private/tls.key
+            - --cookie-secret=$(OAUTH_COOKIE_SECRET)
+            - --pass-user-bearer-token
+            - --skip-provider-button
+            - --approval-prompt=auto
+            - --openshift-ca=/etc/pki/tls/certs/ca-bundle.crt
+            - --openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+            - --openshift-sar={"namespace":"syndesis","resource":"pods","verb":"get"}
+            - --openshift-delegate-urls={"/api/v1/public":{"namespace":"syndesis","resource":"pods","verb":"get"}}
+          env:
+          - name: OAUTH_COOKIE_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: syndesis-oauth-proxy-cookie-secret
+                key: oauthCookieSecret
+          ports:
+          - containerPort: 8443
+            name: public
+            protocol: TCP
+          readinessProbe:
+            httpGet:
+              port: 8443
+              path: /oauth/healthz
+              scheme: HTTPS
+            initialDelaySeconds: 15
+            timeoutSeconds: 10
+          livenessProbe:
+            httpGet:
+              port: 8443
+              path: /oauth/healthz
+              scheme: HTTPS
+            initialDelaySeconds: 15
+            timeoutSeconds: 10
+          volumeMounts:
+          - mountPath: /etc/tls/private
+            name: syndesis-public-oauthproxy-tls
+          resources:
+            limits:
+              memory: 200Mi
+            requests:
+              memory: 20Mi
+        serviceAccountName: syndesis-public-oauthproxy
+        volumes:
+        - name: syndesis-public-oauthproxy-tls
+          secret:
+            secretName: syndesis-public-oauthproxy-tls
+    triggers:
+{{^WithDockerImages}}
+    - imageChangeParams:
+        automatic: true
+        containerNames:
+        - syndesis-public-oauthproxy
+        from:
+          kind: ImageStreamTag
+          name: {{ Images.Support.OAuthProxy }}:{{ Tags.OAuthProxy }}
+          namespace: ${IMAGE_STREAM_NAMESPACE}
+      type: ImageChange
+{{/WithDockerImages}}
+    - type: ConfigChange{{#WithOAuthClient}}
+
+- apiVersion: oauth.openshift.io/v1
+  grantMethod: auto
+  kind: OAuthClient
```

Comment:
@dhirajsb @zregvart is that related to your change or does the import/export also needs an OAuthClient ? then we are running in the same issue with the other PR.

Reply:
Actually it looks like that the API access can also work with the OAuthClient (otherwise it wouldn't be omittted in the non-privilegded template). So I gues this OAuthClient can just be removed here.

Reply:
Removed